src/metabase/public_settings.clj

```diff
@@ -1052,7 +1052,6 @@ See [fonts](../configuring-metabase/fonts.md).")
 (defsetting experimental-search-weight-overrides
   (deferred-tru "Used to override weights used for search ranking")
   :visibility :internal
-  :cache?     false
```

Comment:
This was a bad idea, it caused about 30 extra queries for a legacy search.

TIL that our distributed cache busting is quite smart.